### ISSUE TITLE
Can't Access Log on M1 Mac

### ISSUE THREAD

**Issue by alexmerm** [nextdns/nextdns#666]
**Context**
 - CLI Version: 1.37.10
 - Platform: MacOS 12.3 M1 Mac

Whenever I run `nextdns log` or `sudo nextdns log` I get the response `Error: exit status 1`. I've tried to debug further but I can't get into the log, so its been difficult to figure out what exactly is failing.  Additionally, `nextdns discovered` returns `null`, but I'm not sure if that is a bug or intended functionality.  Everything else works fine and it functions perfectly as a dns server.

**Comment by chr-lei** [comment]
I am having this same issue, Mac OS 12.3, M1 Mac, cli version 1.37.10

**Comment by rs** [comment]
Log management changed in macos. A specific support need to be implemented.

**Comment by robrecord** [comment]
(Same issue on Intel Mac)

**Comment by PY-Chang** [comment]
Same here, my environment is macOS 12.5.1, M1 Pro, nextdns version 1.37.11
Though I encountered this issue on intel Mac before I upgrade to apple silicon

**Comment by lkraider** [comment]
```
% sudo nextdns log
Error: exit status 1
```

**Comment by lkraider** [comment]
```
% uname -v
Darwin Kernel Version 21.6.0: Mon Feb 19 20:24:14 PST 2024; root:xnu-8020.240.18.707.4~1/RELEASE_ARM64_T8101
% nextdns version
nextdns version 1.42.0
% sudo nextdns log
Error: exit status 1
```

**Comment by kurktchiev** [comment]
This was closed and I assume released with 1.43.2, but here it is not working:
```
❯ sudo nextdns version
nextdns version 1.43.3
❯ sudo nextdns log
Password:
Error: exit status 1
```

@rs

**Comment by rs** [comment]
It was moved in a branch for now, as it requires a more complex build pipeline to release. It’s work in progress.
